packages/flutter/test/material/bottom_sheet_theme_test.dart

```diff
@@ -132,6 +132,103 @@ void main() {
     expect(material.elevation, elevation);
     expect(material.shape, shape);
   });
+
+  testWidgets('BottomSheet modalElevation takes priority over elevation - Modal', (WidgetTester tester) async {
+    const double modalElevation = 5.0;
+    const double persistentElevation = 7.0;
+    const BottomSheetThemeData bottomSheetTheme = BottomSheetThemeData(
+      elevation: persistentElevation,
+      modalElevation: modalElevation,
+    );
+    await tester.pumpWidget(bottomSheetWithElevations(bottomSheetTheme));
+    await tester.tap(find.text('Show Modal'));
+    await tester.pumpAndSettle();
+    final Material material = tester.widget<Material>(
+      find.descendant(
+        of: find.byType(BottomSheet),
+        matching: find.byType(Material),
+      ).first,
+    );
+    expect(material.elevation, modalElevation);
+  });
+
+  testWidgets('BottomSheet modalElevation takes priority over elevation - Persistent', (WidgetTester tester) async {
```

Comment:
I feel like this should read `BottomSheet elevation takes priority over modalElevation - Persistent`

because we're expecting that `modalElevation` is ignored in favor of `elevation` here.

Reply:
Done